modules/admin_manual/pages/configuration/integration/ms-teams.adoc

```diff
@@ -0,0 +1,125 @@
+= Microsoft Teams Integration in ownCloud
+:toc: right
+:toclevel: 3
+:appsource-url: https://appsource.microsoft.com/en-us/marketplace/apps?product=teams
+:publish-custom-app-url: https://docs.microsoft.com/en-us/MicrosoftTeams/upload-custom-apps
+:teams-app-setup-policies-url: https://docs.microsoft.com/en-us/microsoftteams/teams-app-setup-policies
+:manage-apps-url: https://docs.microsoft.com/en-us/MicrosoftTeams/manage-apps
+:what-are-tabs-url: https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/what-are-tabs
+
+== Introduction
+
+If you're using Microsoft Teams in your organization or for private purposes, you will likely want to access your ownCloud installation from your Microsoft Teams account. For this purpose, we created the {appsource-url}[Microsoft AppSource] app *ownCloud Generator for Admins* with which you can generate a customized Microsoft Teams app for your users accessing your ownCloud services. Each ownCloud domain accessed requires a separate generated Microsoft Teams app for your users. The generated app will be available in your organization's app catalog.
+
+NOTE: As a prerequisite, the OpenID Connect app is required. If you already have a OpenID Connect configuration made with another service, you have to reconfigure with Microsoft Azure AD, as only one identity provider configuration is allowed.
+
+== Prerequisites
+
+To get this working, you need to install and/or configure four components:
+
+. Microsoft Azure Active Directory
+
+. ownCloud apps:
+.. {oc-marketplace-url}/apps/openidconnect[OpenID Connect] 
+.. https://github.com/owncloud/msteamsbridge[MS-Teams Bridge] 
+// (later available on marketplace or git only?)
```

Comment:
This looks like a future ToDo?

Reply:
Yes, when the app is available on marketplace. it is a remark for us and not rendered.
The doc needs to be out that microsoft sees that we have proper content when we request for publishing the generator app. When all done and the msteamsbridge app is available on the marketplace, we will fix this link.